support-frontend/assets/pages/supporter-plus-landing/twoStepPages/checkoutScaffold.tsx

```diff
@@ -203,53 +220,69 @@ export function SupporterPlusCheckoutScaffold({
 				</FooterWithContents>
 			}
 		>
-			<CheckoutHeading
-				heading={!isPaymentPage && heading}
-				image={
-					!isPaymentPage &&
-					(showUsEoy2023Content ? (
-						<figure css={leftColImageUnitedStates}>
-							<GridImage
-								gridId="supporterPlusLandingUnitedStates"
-								srcSizes={[420]}
-								sizes="420px"
-								imgType="png"
-								altText=""
-							/>
-						</figure>
-					) : (
-						<figure css={leftColImage}>
-							<GridImage
-								gridId="supporterPlusLanding"
-								srcSizes={[500]}
-								sizes="500px"
-								imgType="png"
-								altText=""
-							/>
-						</figure>
-					))
-				}
-				withTopborder={isPaymentPage}
-			>
-				{!isPaymentPage &&
-					(showUsEoy2023Content ? (
-						<p css={subHeading}>
+			{isPaymentPage && (
+				<CheckoutHeading withTopBorder={isPaymentPage}></CheckoutHeading>
+			)}
+
+			{!isPaymentPage &&
+				(showUsEoy2023Content ? (
+					<CheckoutHeading
+						heading={headingUSEoy2023}
+						image={
+							<figure css={leftColImageUnitedStates}>
+								<GridImage
+									gridId="supporterPlusLandingUnitedStates"
+									srcSizes={[420]}
+									sizes="420px"
+									imgType="png"
+									altText=""
+								/>
+							</figure>
+						}
+						withTopBorder={isPaymentPage}
+					>
+						<p css={subHeadingUnitedStates}>
 							We rely on funding from readers, not shareholders or a billionaire
 							owner. Join the more than 250,000 readers in the US whose regular
 							support helps to sustain our journalism.
 						</p>
-					) : (
+					</CheckoutHeading>
+				) : (
+					<CheckoutHeadingImage
+						heading={
+							<figure css={leftColImageHeader}>
+								<Hide from="desktop">
+									<HeadlineImageMobile />
+								</Hide>
+								<Hide until="desktop">
+									<HeadlineImageDesktop />
+								</Hide>
+							</figure>
```

Comment:
This will not include the heading for both breakpoints in the `HTML` as they are SVGs. This means we load ~27kb of unused HTML (uncompressed). Not a problem I don't think but something worth considering as we move into being able to serve SVGs.

Reply:
Yes, seen hide used elsewhere like this though cant help thinking there is a better way. It works ok for now would say assuming thats ok.